BF: Remove csv_paths array from download

tdooner opened the pull request:
When called with `--skip-download`, as we do, the `csv_paths` array is
uninitialized, resulting in an error.

Instead, we can just dynamically construct the array of CSVs using a
glob to match the file pattern `netfile_*_*_cal201_export.csv` which
will need to match the downloaded filename.

Fixes #142.

tdooner committed:
BF: Remove csv_paths array from download

When called with `--skip-download`, as we do, the `csv_paths` array is
uninitialized, resulting in an error.

Instead, we can just dynamically construct the array of CSVs using a
glob to match the file pattern `netfile_*_*_cal201_export.csv` which
will need to match the downloaded filename.

bcipolli commented:
I believe this will break selective loading ... if many `csv`s exist but say one new `agency` is downloaded to be imported.

tdooner committed:
BF: Compute this.agencies and this.years in handle method

Since the --agencies and --years flags apply to multiple parts of the
downloadnetfilerawdata command, I hoisted their computation logic into
the handle method.

tdooner commented:
Drats, you are right. I adjusted the logic in this commit to instead compute `this.agencies` and `this.years` in the handle method, which should apply equally to the `download` and `combine` steps.

Commit:
STY: remove unused import

bcipolli commented:
:+1: LGTM. Fixed a small PEP8 issue.. @tdooner LMK if you want help setting up `flake8` for your editor at any point.

tdooner commented:
Yeah, I use vim/syntastic, so I'm not sure why it's not working. I'll figure it out at some point ;)
